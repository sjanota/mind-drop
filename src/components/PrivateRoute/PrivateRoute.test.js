import React from 'react';
import renderer from 'react-test-renderer';
import PrivateRoute from "./PrivateRoute";
import {Auth0Context} from "../../react-auth0-wrapper";
import {MemoryRouter as Router} from 'react-router-dom'

it('renders without crashing', () => {
  const tree = renderer
    .create(<Router><Auth0Context.Provider value={{isAuthenticated: true}}><PrivateRoute component={"div"}/></Auth0Context.Provider></Router>)
    .toJSON();
  expect(tree).toMatchSnapshot()
});
