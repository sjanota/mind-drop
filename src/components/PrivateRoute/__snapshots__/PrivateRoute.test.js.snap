// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  history={
    Object {
      "action": "POP",
      "block": [Function],
      "canGo": [Function],
      "createHref": [Function],
      "entries": Array [
        Object {
          "hash": "",
          "key": "vhwgfv",
          "pathname": "/",
          "search": "",
          "state": undefined,
        },
      ],
      "go": [Function],
      "goBack": [Function],
      "goForward": [Function],
      "index": 0,
      "length": 1,
      "listen": [Function],
      "location": Object {
        "hash": "",
        "key": "vhwgfv",
        "pathname": "/",
        "search": "",
        "state": undefined,
      },
      "push": [Function],
      "replace": [Function],
    }
  }
  location={
    Object {
      "hash": "",
      "key": "vhwgfv",
      "pathname": "/",
      "search": "",
      "state": undefined,
    }
  }
  match={
    Object {
      "isExact": true,
      "params": Object {},
      "path": "/",
      "url": "/",
    }
  }
/>
`;
